Add gcp blueprint

Jessiebr opened the pull request.

Jessiebr committed:
Add GCP blueprint

Jessiebr committed:
Add GCP blueprint

Jessiebr committed:
Add GCP blueprint

Jessiebr committed:
Add GCP blueprint

Jessiebr committed:
fix blueprint

Jessiebr committed:
fix blueprint

Jessiebr committed:
fix blueprint

EarthmanT committed:
adding bad status line troubleshooting

EarthmanT committed:
updating types.yaml 4.1

Commit:
added blog post tutorial links

EarthmanT committed:
4.1 updates

Jessiebr committed:
Fixed secrets and nodecellar url

Commit:
fixed typo

Commit:
some changes

Commit:
changed location of deployment outputs

Jessiebr committed:
fix comments

EarthmanT committed:
updating script to alway be created

EarthmanT committed:
adding circle.yml

Commit:
added clearer info

Commit:
fixed

Commit:
more edits

diaperrash committed:
Updating formatting.

EarthmanT committed:
adding awssdk plugin

EarthmanT committed:
updating with more plugins

EarthmanT committed:
Merge pull request #5 from cloudify-examples/4.1

4.1

Commit:
fixes

Jessiebr committed:
Update GCP blueprint to 4.1

Jessiebr committed:
Update GCP blueprint to 4.1

EarthmanT commented on gcp-blueprint.yaml:
```diff
@@ -0,0 +1,262 @@
+
+inputs:
+
+  gcp_auth_file:
```
We need to find a different way of providing authentication details. A file is not secure and not one of our "best practices".

EarthmanT commented on examples/gcp-simple-blueprint.yaml:
```diff
@@ -0,0 +1,29 @@
+tosca_definitions_version: cloudify_dsl_1_3
```
This file does not belong in the repo.

EarthmanT commented on gcp-blueprint.yaml:
```diff
@@ -0,0 +1,262 @@
+
+  1-Instructions:
+    value: { get_attribute: [ ManagerSetup, final_path ] }
+  2-Demo:
```
These instructions have changed.

Commit:
updating for 4.1.1

Commit:
updated links

EarthmanT committed:
Merge branch 'latest' into 4.1.1

Commit:
updating aws plugin version

Commit:
updating aws plugin version

EarthmanT committed:
Merge pull request #9 from cloudify-examples/4.1.1

4.1.1

Commit:
add troubleshooting scenario

Commit:
added some clarity

Commit:
fixed uninstall command

Commit:
removed line

Commit:
edited IP addresses and moved text

Commit:
Making AWS more user friendly

changing region to a lower demand region (i hope)
replacing need for nat instance
updating readme
adding awssdk nat gateway

Commit:
adding aws ami check

Commit:
making changes

Commit:
updating readme
